evaluation.py

```diff
@@ -48,11 +48,10 @@ def predict_heatmaps(self, X):
         predict_min = np.min(predict_heatmaps)
         predict_var = np.var(predict_heatmaps.flatten())
         print('Mean: {:0.6e}\t Max: {:e}\t Min: {:e}\t Variance: {:e}'.format(predict_mean, predict_max, predict_min, predict_var))
+        # TODO: is this how we want to normalize output? Should we instead enforce model output to be normalized?
```

Comment:
This is temporary for now. I'll add a sigmoid layer at the end of each model output and see how it does.